### ISSUE TITLE
Want to discuss a behaviour of VKD3D that performed quite different between running on Windows OS and SteamOS (on Steamdeck)

### ISSUE THREAD

**Issue by ycyleo** [HansKristian-Work/vkd3d-proton#2057]
### System information
- GPU: SteamDeck APU
- Wine version: 9.14
- VKD3D-Proton version: 2.12 

Hi guys, as the title, I have aquestion about a behaviour of vkd3d, I may not able to describe that accurately, sorry about that. Also, all I mention in below was happen on a Steamdeck.

So basically, I had a game that quite demanding to run it, when I using vkd3d to run the game on windows, 
it was run at around 40-55 fps, however, the game seem set target fps to 60fps, so the game will run in slow motion.

When the game run on SteamOS with lutris, the game is running much better at around 50-60fps but sometime will still drop,
and here is the main thing I want to discuss, about "frameskipping", so the gamescope always show the game running 60fps but sometime you can actually feel that it was frameskipping, not smooth 60fps, but the game speed will be normal and not running like slow motion.

Especially when I control the GPU run at lower mhz(like 800-1000mhz), the game will be more frameskipping but the game speed can still maintain in normal.

May I know if the frameskipping will only happen when vkd3d run on SteamOS(or Linux base OS)?
Or that actually not a frameskipping, maybe that is something else but I get it wrong.

Much appreciate if anyone can tell me any details of this.

**Comment by doitsujin** [comment]
Sounds like a game logic thing, vkd3d-proton does not affect that in any way.

**Comment by ycyleo** [comment]
Oh I see, so that can't be explainable :(
but still feel weird that why this behaviour only happen on SteamOS

**Comment by HansKristian-Work** [comment]
It's possible the game is using Present interval. In 2.12, we would duplicate presents, which can explain the 60 fps metric. Try master, where we removed that.

**Comment by ycyleo** [comment]
Ah, actually on windows side I only use 2.4 (or newer until 2.9), other than that, newer will make the game slow down more more without reason or even break the game.  So let say if I just want to this kind of old specific version, is there any environment variable or dxvk.conf command can help?

**Comment by HansKristian-Work** [comment]
No. Which game is this? If there is a regression, it should be looked at.

**Comment by ycyleo** [comment]
Actually the game is a private project from someone, so I worry that if I share the log, then it may expose something that the creator not want to show.

Anyway, much appreciate for some hints on this case.

**Comment by Blisto91** [comment]
@ycyleo Was this question or issue solved?
